### DISCUSSION TITLE
Unable to add session

### DISCUSSION THREAD

**Discussion by colinsolid** [lightninglabs/lightning-terminal#722] [Troubleshooting]
Hi, 
I'm getting this error 

UNABLE TO ADD SESSION
lit is not ready for: /litrpc.Sessions/Addsession.

Any insights on this? Thanks

**Comment by pulpf1** [comment]
hi, i have the same problem

**Comment by levmi** [comment]
What version of litd and lnd?

**Reply by pulpf1** [reply]
0.12.4-alpha this is the version in my runcitadel node with an RPI5

**Comment by tomesdesign** [comment]
Same problem here. Why is  this issue closed?

**Reply by guggero** [reply]
Because a restart likely solved it for the user. If not, please provide logs and additional details.
